client/core/bond.go

```diff
@@ -424,6 +460,26 @@ func (c *Core) rotateBonds(ctx context.Context) {
 					c.log.Errorf("Unable to post bond: %v", err)
 				} // else it's now in pendingBonds
 			}
+		} // mustPost
+	} // range c.dexConnections
+
+	for assetID, reserveDelta := range netReserveDeltas {
+		if reserveDelta != 0 { // net adjustment across all dexConnections
+			wallet, err := c.connectedWallet(assetID)
+			if err != nil { // we grabbed it above so shouldn't happen
+				c.log.Errorf("%v wallet not available for bond: %v", unbip(assetID), err)
+				continue
+			}
+			c.log.Infof("Updating bond reserves by %s (automatic tier change adjustments)",
+				wallet.amtStringSigned(reserveDelta))
+			wallet.ReserveBondFunds(reserveDelta, false)
```

Comment:
Is it possible for `rotateBonds` to run concurrently?

Reply:
Should only be run from the loop in `watchBonds`.  Will document that it's not to be run willy nilly.